dynatrace/build.gradle

```diff
@@ -20,27 +20,30 @@ plugins {
 }
 
 group = "com.dynatrace.opentelemetry.metric"
-version = "0.4.0-alpha"
+version = "0.5.0"
```

Comment:
Hmm, should we be going to `1.0.0` for this? :thinking:

Reply:
Sounds good. We'll do the same in the .NET exporter. I will mark this as resolved.